.circleci/src/jobs/check_web.yml

```diff
@@ -10,15 +11,28 @@ steps:
   - checkout
   - check_config
   - restore_yarn_cache
-  - lint:
-      directory: web
   - lint:
       directory: api-client
+  - lint:
+      directory: build-configs
+  - lint:
+      directory: web-ts
+  - lint:
+      directory: translations
+  - run:
+      name: TS check
+      command: yarn workspace api-client ts:check
+  - run:
+      name: TS check
+      command: yarn workspace translations ts:check
+  - run:
+      name: TS check
+      command: yarn workspace build-configs ts:check
   - run:
-      name: Check flow
-      command: yarn workspace web flow:check-now --max-workers ${TOTAL_CPUS}
+      name: TS check
+      command: yarn workspace web-ts ts:check
```

Comment:
I think we should remove this check. It will be checked during compilation later. This is just checking it twice and will increase build time.

Reply:
But don't we just check with our webpack tsconfig? So I think one could error while one does not? Or am I wrong here?

Reply:
No, the webpack ts config is for the webpack config which is written in ts.